askama_parser/src/expr.rs

```diff
@@ -927,6 +927,27 @@ impl<'a> Suffix<'a> {
             }
         }
 
+        fn lifetime<'a>(i: &mut InputStream<'a>) -> ParseResult<'a, ()> {
+            // Before the 2021 edition, we can have whitespace characters between "r#" and the
+            // identifier so we allow it here.
+            let start = *i;
+            '\''.parse_next(i)?;
+            if opt(alt((
+                (keyword("r#"), opt(skip_ws0), identifier).map(|(_, _, _)| ()),
```

Comment:
I don't think this code can ever match. A `#` cannot be part of a keyword. I guess this should be `"r#"`.

Can you please add a test that contains raw lifetimes that get parsed correctly? Something as simple as `macro_rules! test { ('r#ignore_me) => { "ok" } }` → `{{ test!('r#ignore_me) }}` → `assert_eq!(.., "ok")`?